app/labor/badge_rewarder.rb

```diff
@@ -10,12 +10,13 @@ module BadgeRewarder
   MINIMUM_QUALITY = -25
 
   def self.award_yearly_club_badges
-    (1..3).each do |i|
+    total_years = Time.current.year - ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"].to_i
```

Comment:
Decided while I was here to make this a little more dynamic

Reply:
Looks good to me. This is notorious off-by-one territory so I had to count really carefully. 😄